plugins/woocommerce/src/Internal/Admin/Events.php

```diff
@@ -158,7 +158,7 @@ public function do_wc_admin_daily() {
 
 		if ( $this->is_remote_inbox_notifications_enabled() ) {
 			DataSourcePoller::get_instance()->read_specs_from_data_sources();
-			RemoteInboxNotificationsEngine::run();
+			RemoteInboxNotificationsEngine::possibly_run();
```

Comment:
Also updated it here, as this appears to be run when WooCommerce is initially installed/enabled.